Source_Code/Common/Messages/ActionRequests/AccuseRequest.java

```diff
@@ -0,0 +1,20 @@
+package Common.Messages.ActionRequests;
+
+import Common.SolutionHand;
+
+public class AccuseRequest extends ActionRequest
+{
+    public String PlayerName;
```

Comment:
Added PlayerName to ActionRequest, so we don't have to have it here

Reply:
As I was writing these 84 times I was like "inheritence should have taken care of this..." but was too lazy to check.